build/build.ps1

```diff
@@ -0,0 +1,63 @@
+param (
```

Comment:
This file also moved.  Most of the rest of this PR is still in the foundationdb repo.  @ammolitor, is that intentional?